docs/framework/tutorial/shard.md

```diff
@@ -365,4 +367,112 @@ Looking at the `content/` directory, there are actually two files in it:
+
+We'll merge featured sets with only one post into one generic "Blog Posts" set for the purposes of this walkthrough, and create a featured set from a few free-form tags that have lots of posts associated with them.
+Normally we would not do this, since we'd have more content and the number of posts in each set will change anyway as new content rotates in and out, but for this purpose it's nice not to have sets with lone articles in them.
+
+The featured sets will be:
+
+| Set name               | Tag(s)              | Number of articles\*     |
+| ---------------------- | ------------------- | ------------------------ |
+| About Creative Commons | `About CC`          | 13                       |
+| Around the World       | `Global affiliates`, plus all the tags that mention a specific country, city, or language | 11 |
+| Policy, Advocacy & Copyright Reform | `Policy / advocacy / copyright reform` | 9 |
+| Events                 | `Events`            | 7                        |
+| Blog Posts             | `Education / OER`, `Open access`, `Weblog` | 4 |
+| Interviews             | `tag:cctalkswith`   | 3                        |
+
+<sup>\*</sup> At least, in the hatch at the time this tutorial was written.
+
+We'll also choose some of the free-form tags to be non-featured sets.
+For these it doesn't matter if they have only a lone post associated with them.
+They won't show up on the main menu, only as tags at the bottom of articles.
+For example, we'll create a "Copyright Reform" set covering the tags `tag:copyright reform`, `tag:eu copyright reform`, and `tag:#fixcopyright`.
+
+## Creating a permanent shard for the sets ##
+
+We'll write a Basin manifest for the permanent shard and use Basin to create it.
+The manifest looks something like this:
+```json
+{
+    "sets": [
+        {
+            "title": "About Creative Commons",
+            "featured": true,
+            "childTags": [
+                "About CC"
+            ]
+        },
+
+        ...
+
+        {
+            "title": "Copyright Reform",
+            "featured": false,
+            "childTags": [
+                "tag:#fixcopyright",
+                "tag:copyright reform",
+                "tag:eu copyright reform"
+            ]
+        },
+
+        ...
+    ]
+}
+```
+(This shows one example each of a featured set and a non-featured set.)
+
+Compare this to how we created a shard [earlier](#creating-a-shard-with-basin).
+To create this shard, we'll use the Basin tool itself instead of `basin-hatch`.
+The `basin-hatch` tool uses Basin under the hood.
+The Basin manifest is the lowest-level description of what goes into a shard, and if you are writing more complicated apps you can write a tool similar to `basin-hatch` that converts from some higher-level format into a Basin manifest and uses Basin under the hood to create the shard.
+
+Using Basin to convert the manifest into a shard is simple:
+```bash
+basin sets.json sets.shard
```

Comment:
If we end up just using basin directly, then it might not make sense to depend on hatch_sets.json file API.